components/channel_layout/center_channel/center_channel.tsx

```diff
@@ -58,6 +70,27 @@ export default class CenterChannel extends React.PureComponent<Props, State> {
         return {lastReturnTo: nextProps.location.pathname};
     }
 
+    async componentDidMount() {
+        const {actions, showNextSteps, showNextStepsTips, isOnboardingHidden, userExistedForSomeTime, currentUserId} = this.props;
+        await actions.getProfiles();
+
+        // We call getUser want to make sure we have the current user's profile into the profiles state
+        await actions.getUser(currentUserId);
+        if ((showNextSteps || showNextStepsTips) && !isOnboardingHidden && !userExistedForSomeTime) {
+            actions.setShowNextStepsView(true);
+        }
```

Comment:
It seems that if they change teams, this will make it show again even though they navigated away from it and it should only be showing in the sticky menu in the LHS

Reply:
I have tested and switching teams doesn't make it show again

Reply:
This makes sure it doesn't show 
```js
    componentDidUpdate(prevProps: Props) {
        const {location, showNextStepsEphemeral, actions, currentUserId} = this.props;
        if (location.pathname !== prevProps.location.pathname && showNextStepsEphemeral) {
            actions.setShowNextStepsView(false);
        }

        // We call getUser want to make sure we have the current user's profile into the profiles state
        actions.getUser(currentUserId);
    }

```